content/build/hydrobuild.md

```diff
@@ -61,106 +61,57 @@ To start building with Hydrobuild, you must create a new builder using the
+architectures by default. This gives you a high-performance build cluster for
+building multi-platform images natively.
 
 You can omit the `cloud://` protocol prefix from the endpoint when you create a
 builder using the `cloud` driver.
 
-### Create a single-platform builder
-
-To create a `linux/amd64` builder:
+### Create a builder
 
 1. Sign in to your Docker ID using the Docker Desktop UI or the `docker login`
    command.
 
 2. Create a builder that uses the `cloud` driver.
 
    ```console
-   $ docker buildx create --driver cloud --name hydrobuild \
-     --platform linux/amd64 \
-     <org>/default_linux-amd64
+   $ docker buildx create --driver cloud <org>/default
    ```
 
    Replace `<org>` with the Docker organization.
 
-### Create a multi-platform builder
-
-To create a builder with support for native `linux/amd64` and `linux/arm64`
-builds:
-
-1. Sign in to your Docker ID using the Docker Desktop UI or the `docker login`
-   command.
-
-2. Create a `linux/amd64` builder that uses the `cloud` driver.
+   This creates a builder named `cloud-<org>-default`.
 
-   ```console
-   $ docker buildx create --driver cloud --name hydrobuild \
-     --platform linux/amd64 \
-     <org>/default_linux-amd64
-   ```
-
-   Replace `<org>` with the Docker organization.
-
-3. Create a `linux/arm64` builder and append it to the `hydrobuild` builder you
-   just created.
-
-   ```console
-   $ docker buildx create --append --name hydrobuild \
-     --platform linux/arm64 \
-     <org>/default_linux-arm64
-   ```
-
-   `<org>` should be the same as for first builder, but this time
-   use `linux-arm64` for the platform suffix.
-
-## Use Hydrobuild
-
-To build your applications with Hydrobuild, you can:
-
-- [Use the Docker CLI](#cli) to build from your local development machine
-- [Use GitHub Actions](#github-actions) to build with Hydrobuild in CI
-
-### CLI
+## Use Hydrobuild from the CLI
 
 To run a build using Hydrobuild, invoke a build command and specify the
 name of the builder using the `--builder` flag.
 
 ```console
-$ docker buildx build --builder hydrobuild --tag myorg/some-tag .
+$ docker buildx build --builder cloud-<org>-default --tag <org>/<image> .
 ```
 
 > **Note**
 >
-> Specifying `--tag` ensures that the build result gets exported to your local
-> image store when the build finishes. If you want to download the results from
-> Hydrobuild without specifying a tag, you must pass the `--load` flag.
+> Building with `--tag` loads the build result to the local image store
+> automatically when the build finishes. To build without a tag and load the
+> result, you must pass the `--load` flag.
 >
 > If you use the containerd image store, you must always pass `--load` to
 > download the results, even if you build with a tag.
 
-If you created a [multi-platform builder](#create-a-multi-platform-builder),
-you can build multi-platform images using the `--platform` flag:
+To build a multi-platform image and push it to a registry:
```

Comment:
Not sure if we should specify that pushing to a registry is the only supported way to build multi platform images?